packages/tupaia-web/src/features/Visuals/Matrix/parseData.tsx

```diff
@@ -2,10 +2,15 @@
  * Tupaia
  *  Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
  */
+import { generatePath } from 'react-router-dom';
 import { MatrixColumnType, MatrixRowType, SearchFilter } from '@tupaia/ui-components';
 import { formatDataValueByType } from '@tupaia/utils';
-import { MatrixConfig, MatrixReportColumn, MatrixReportRow } from '@tupaia/types';
-import { URL_SEARCH_PARAMS } from '../../../constants';
+import { MatrixConfig, MatrixReportColumn, MatrixReportRow, MatrixEntityCell } from '@tupaia/types';
+import { ROUTE_STRUCTURE, URL_SEARCH_PARAMS } from '../../../constants';
+
+function isMatrixEntityCell(cell: unknown): cell is MatrixEntityCell {
```

Comment:
I wonder if this should go in `types` since it is used here and in the report server?